Tools/speech_recognizer/GSSpeechRecognizer.m

```diff
@@ -0,0 +1,101 @@
+#import "GSSpeechRecognizer.h"
+
+static GSSpeechRecognitionServer *server;
+static int clients;
+
+@interface GSSpeechRecognizer (Private)
++ (void)connectionDied: (NSNotification*)aNotification;
+@end
+
+@implementation GSSpeechRecognizer
++ (void)initialize
+{
+  server = [GSSpeechRecognitionServer sharedServer];
+
+  RETAIN(server);
+  if (server != nil)
+    {
+      clients++;
```

Comment:
That would allow for exactly one client.

Reply:
Fixed